### PR TITLE
Jobs: Use class selector instead of id

### PR DESCRIPTION
**Pull request by Echoring**
Once discussed in #2014 
Retain id for easier access when special needed.

### CODE REVIEW

File: ui/jobs/jobs.js [hunk 1]
```diff
@@ -1601,6 +1603,7 @@ class Bars {
       container = document.createElement('div');
       container.id = id;
       document.getElementById('bars').appendChild(container);
+      container.classList.add('procbox');
```
Comment (MaikoTan) [inline ui/jobs/jobs.js:1606]:
```suggestion
      container.classList.add('proc-box');
```

add a '-' is better

File: ui/jobs/jobs.js [hunk 2]
```diff
@@ -1614,9 +1617,10 @@ class Bars {
     timerBox.hideafter = '';
     timerBox.roundupthreshold = false;
     timerBox.valuescale = options.scale ? options.scale : 1;
-    if (options.id)
+    if (options.id) {
       timerBox.id = options.id;
-
+      timerBox.classList.add('timerbox');
```
Comment (MaikoTan) [inline ui/jobs/jobs.js:1622]:
```suggestion
      timerBox.classList.add('timer-box');
```

also here

File: ui/jobs/jobs.js [hunk 3]
```diff
@@ -1628,6 +1632,7 @@ class Bars {
     let timer = document.createElement('timer-bar');
     container.appendChild(timerDiv);
     timerDiv.appendChild(timer);
+    timer.classList.add('timerbar');
```
Comment (MaikoTan) [inline ui/jobs/jobs.js:1635]:
```suggestion
    timer.classList.add('timer-bar');
```

and here

**Comment by quisquous** [comment]
Thanks! This is much improved!

**Pull request merged**